signup/models.py

```diff
@@ -235,6 +235,8 @@ class Contact(models.Model):
         " followed by last name)"))
     nick_name = models.CharField(_("Nick name"), max_length=60, blank=True,
         help_text=_("Short casual name used to address the contact"))
+    picture = models.CharField(_("Profile picture"), max_length=300, blank=True,
```

Comment:
I believe Django has a `models.URLField` that could be used here. Maybe. If it accepts s3:// and file:// prefixes.

Reply:
That's true, but I'm not actually storing the string in a valid URL format, the picture string is a concatenation of region, bucket and a file name. I can store it as a valid URL, this way we won't need the `picture_url` method.